frontend/ui/lib/widgets/create_trip_page_widget.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:tripmeout/widgets/create_trip_widget.dart';
+import 'package:tripmeout/widgets/map_widget.dart';
+
+class CreateTripPageWidget extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+        child: FittedBox(
+      child: Column(
+        children: <Widget>[
+          Row(
+            children: <Widget>[
```

Comment:
This looks like a good start for getting the layout correct. Consider moving some of the nested elements into their own widgets to make this easier to compose. When you start populating this with real data and adding behavior, adding elements in-line will quickly become unwieldy.

Reply:
I agree with Ben, but this is actually much trickier with the creation page than with other pages, so I'm ok with it for now.

Reply:
👍